katas/codingmeetup6_samelanguage.js

```diff
@@ -0,0 +1,5 @@
+//From https://www.codewars.com/kata/58287977ef8d4451f90001a0
+
+function isSameLanguage(list) {
+  return list.every(function(x) {return x.language === list[0].language})
```

Comment:
Inline code is difficult for humans to parse.  Consider whitespace (especially new lines!) in order to make this more readable.  Variable name `x` should be renamed to be more intention revealing.